src/msw/textctrl.cpp

```diff
@@ -812,6 +832,39 @@ WXDWORD wxTextCtrl::MSWGetStyle(long style, WXDWORD *exstyle) const
     return msStyle;
 }
 
+#if wxUSE_RICHEDIT && wxUSE_SPELLCHECK
+
+bool wxTextCtrl::EnableProofCheck(bool enable, const wxTextProofOptions& options)
```

Comment:
AFAICS this is not used here currently:
```suggestion
bool wxTextCtrl::EnableProofCheck(bool enable, const wxTextProofOptions& WXUNUSED(options))
```